### DISCUSSION TITLE
what is Added WorkloadMode selection to Logging. mean?

### DISCUSSION THREAD

**Discussion by 13567436138** [istio/istio#49441] [Q&A] [answered]
https://istio.io/latest/news/releases/1.14.x/announcing-1.14/change-notes/

Telemetry
Added the implementation of the OpenTelemetry access log.

Added environment variable support at Wasm extension via VM configuration in WasmPlugin API.

Added WorkloadMode selection to Logging.

Added support for tracing WorkloadMode in Telemetry API. This will allow customization of tracing behavior based on traffic direction.

Added initial flag-protected support for exporting canonical service labels for ServiceEntry resources with a location of MESH_EXTERNAL.

**Comment by zirain** [comment, marked as answer]
https://istio.io/latest/docs/reference/config/telemetry/#AccessLogging-LogSelector
